### ISSUE TITLE
base image can't be store under specified path

### ISSUE THREAD

**Issue by gtycherry** [mjhea0/cypress-visual-regression#9]
When integrated cypress-visual-regression with cypress, and run command:
cypress run type=base --config screenshotsFolder=cypress/results/snapshots/base
the base image can't be placed under path 'cypress/results/snapshots/base'.
Can anybody give some suggestion?

**Comment by damiano-carradori** [comment]
I've just try to run it, but I didn't get any error and the images are inside the `cypress/results/snapshots/base` folder.

Can you attach the error that you receive?

**Comment by mjhea0** [comment]
closing in favor of https://github.com/mjhea0/cypress-visual-regression/issues/13

**Comment by jbutko** [comment]
I had a very similar issue that base snapshots were not created in specified folder which I have passed via `--config screenshotsFolder=...` flag in command line. I figured out that I did not passed `config` flag correctly. At first I did following:
```
.\node_modules\.bin\cypress run --env type=base --config screenshotsFolder=cypress/snapshots/base --config testFiles=\"**/*regression-tests.js\"
```
See I duplicated `--config` option flag because I needed to specify file where regression tests were placed. And that's the issue. Based on [cypress command line documentation](https://docs.cypress.io/guides/references/configuration.html#Command-Line) if you are passing multiple options they need to be separated with comma. So the correct command should be:
```
.\node_modules\.bin\cypress run --env type=base --config screenshotsFolder=cypress/snapshots/base,testFiles=\"**/*regression-tests.js\"
```
After that fix all worked like a charm. Maybe this helps somebody else facing similar issue.
